webpack.config.js

```diff
@@ -254,25 +253,14 @@ if ( process.env.DASHBOARD ) {
 	webpackConfig.plugins.unshift( new DashboardPlugin() );
 }
 
-if ( process.env.WEBPACK_OUTPUT_JSON ) {
+if ( ! isDevelopment ) {
 	webpackConfig.devtool = 'cheap-module-source-map';
 	webpackConfig.plugins.push(
-		new webpack.optimize.UglifyJsPlugin( {
-			minimize: true,
-			compress: {
-				warnings: false,
-				conditionals: true,
-				unused: true,
-				comparisons: true,
-				sequences: true,
-				dead_code: true,
-				evaluate: true,
-				if_return: true,
-				join_vars: true,
-				negate_iife: false,
-				screw_ie8: true,
-			},
-			sourceMap: true,
+		new UglifyJsPlugin( {
+			cache: true,
+			parallel: true,
+			uglifyOptions: { ecma: 5 },
+			sourceMap: false,
```

Comment:
Is this supposed to be equivalent? Why replace `sourceMap: true` with `false` here?

Reply:
prior, we only really used the plugin to minimize the source for the bundle analyzer. I think that required sourcemaps to work correctly...

@samouri might check and see if `npm run analyze-bundles` still works with source maps disabled here?

Reply:
> Is this supposed to be equivalent?

so the old granular settings have all been moved to live within the new `uglifyOptions` key. 
In the past we had been setting flags individually but I figured if the defaults are better (which they are by 30kb), then lets just stick with those.

> Why replace sourceMap: true with false here?

good question! before this pr sourceMap for production was always false. it was only true when building for `analyze-bundles` or when building in development.  I'm actually in favor of enabling it in all of our environments, but I'd rather leave that for a tiny followup pr since that may be controversial.

sourcemaps:
-  make builds much (will need to measure how much) slower
- make debugging possible
-  are pretty large but only download if devtools are open (regular users won't need to download them).